### ISSUE TITLE
Troubleshooting the build process

### ISSUE THREAD

**Issue by MarcoMeter** [hanzi/libmgba-py#1]
Thanks for sharing a streamlined approach to build mGBA with python bindings!

I'm currently running into an issue during the building process leveraging `build_win64.bat`. My platform is Windows 10. Git, Visual Studio Community 2022, and Python are installed.

During executing `cmake -DCMAKE_TOOLCHAIN_FILE=./vcpkg/scripts/buildsystems/vcpkg.cmake -DBUILD_SHARED=ON -DBUILD_STATIC=ON ..` this output is salient, but I'm not sure whether this poses an issue for the later errors:

```
CMake Error in src/platform/qt/CMakeLists.txt:
  Evaluation file to be written multiple times with different content.  This
  is generally caused by the content evaluating the configuration type,
  language, or location of object files:

   D:/Repositories/libmgba-py/mgba-src/build/qt/updater.qrc
```

The next issue emerges when executing `msbuild ALL_BUILD.vcxproj /property:Configuration=Release`, but can't really trace the actual exception or error:

```
Done Building Project "D:\Repositories\libmgba-py\mgba-src\build\updater-stub.vcxproj" (default targets).

Done Building Project "D:\Repositories\libmgba-py\mgba-src\build\ALL_BUILD.vcxproj" (default targets) -- FAILED.


Build FAILED.
```

I guess the last step `python setup.py build --build-lib build/win64` fails because of the former issues.

```
D:\Repositories\libmgba-py\_builder_cdef.h(64): fatal error C1083: Cannot open include file: 'mgba-util/png-io.h': No such file or directory
Traceback (most recent call last):
  File "setup.py", line 36, in <module>
    setup(
  File "C:\ProgramData\Anaconda3\lib\site-packages\setuptools\__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "C:\ProgramData\Anaconda3\lib\distutils\core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\setuptools\dist.py", line 423, in __init__
    _Distribution.__init__(self, {
  File "C:\ProgramData\Anaconda3\lib\distutils\dist.py", line 292, in __init__
    self.finalize_options()
  File "C:\ProgramData\Anaconda3\lib\site-packages\setuptools\dist.py", line 695, in finalize_options
    ep(self)
  File "C:\ProgramData\Anaconda3\lib\site-packages\setuptools\dist.py", line 702, in _finalize_setup_keywords
    ep.load()(self, ep.name, value)
  File "C:\ProgramData\Anaconda3\lib\site-packages\cffi\setuptools_ext.py", line 219, in cffi_modules
    add_cffi_module(dist, cffi_module)
  File "C:\ProgramData\Anaconda3\lib\site-packages\cffi\setuptools_ext.py", line 49, in add_cffi_module
    execfile(build_file_name, mod_vars)
  File "C:\ProgramData\Anaconda3\lib\site-packages\cffi\setuptools_ext.py", line 25, in execfile
    exec(code, glob, glob)
  File "_builder.py", line 35, in <module>
    subprocess.check_call(preprocessor_command + [str(path_to_libmgba_py / "_builder_cdef.h")])
  File "C:\ProgramData\Anaconda3\lib\subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cl', '/P', '/DWIN32_LEAN_AND_MEAN', '/ID:\\Repositories\\libmgba-py', '/ID:\\Repositories\\libmgba-py\\..\\mgba\\build\\include', '/ID:\\Repositories\\libmgba-py\\..\\mgba\\include', '/ID:\\Repositories\\libmgba-py\\..\\mgba\\src', '/ID:\\Repositories\\libmgba-py\\..\\mgba\\build\\vcpkg\\installed\\x64-windows\\include', 'D:\\Repositories\\libmgba-py\\_builder_cdef.h']' returned non-zero exit status 2.
mgba-src\build\Release\avcodec-60.dll
The system cannot find the path specified.
```

This is the full output of the building process:
[log.txt](https://github.com/hanzi/libmgba-py/files/13875870/log.txt)

Is there more information that I could contribute to this matter?
I'm thankful for any advice =)

**Comment by hanzi** [comment]
**tl;dr** There was a bug that I've just fixed; checking out the current master should resolve that.

There was a wrong path in the Python extension build script. `build_win64.bat` checks out the mGBA repository to `./mgba-src/` and compiles it... and the Python script didn't try to use that, but instead looked at `../mgba/`.

On your system, this would resolve to `D:\Repositories\mgba\` -- and I guess you've checked out the repository to that exact location yourself, by chance. And this is probably either the master branch or v0.10.3, where the `png-io.h` header has been moved to a different place.

I have fixed that in https://github.com/hanzi/libmgba-py/commit/a2a15fd6f8594f08c118a0a5369a8df03db484f7. Let me know if there are any more issues with that, and thank you for reporting the issue to begin with! 🙂

**Comment by MarcoMeter** [comment]
Thanks for your immediate return!

This resolved my issue =)
